user_guide_src/source/helpers/html_helper.rst

```diff
@@ -31,10 +30,10 @@ The following functions are available:
     :param  string|array  $src:  Image source URI, or array of attributes and values
     :param  bool    $indexPage:  Whether to treat ``$src`` as a routed URI string
     :param  mixed   $attributes: Additional HTML attributes
-    :returns:   HTML image tag
+    :returns:   HTML image element
     :rtype: string
 
-    Lets you create HTML ``<img />`` tags. The first parameter contains the
+    Lets you create HTML ``<img />`` elements. The first parameter contains the
```

Comment:
Is `<img>` instead of `<img />` better?

Reply:
@kenjis Yes, they are all better. But that was the first part of old PR. In the second and third parts those fixes were  coming.